dotnet-sdk/9.0/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM mcr.microsoft.com/dotnet/sdk:9.0.101-bookwork-slim
```

Comment:
should this be `bookworm` instead of `bookwork`?

Reply:
fixed in the manifest